systemd/jenkins.service

```diff
@@ -0,0 +1,142 @@
+#
+# This file is managed by systemd(1). Do NOT edit this file manually!
+# To override these settings, run:
+#
+#     systemctl edit @@ARTIFACTNAME@@
+#
+# For more information about drop-in files, see:
+#
+#     https://www.freedesktop.org/software/systemd/man/systemd.unit.html
+#
+
+[Unit]
+Description=Jenkins Continuous Integration Server
+Requires=network.target
```

Comment:
Dependency may be weaker - `Wants=`
https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Wants=

Reply:
> Dependency may be weaker

It may indeed.

Reply:
By reducing dependency requirements you will reduce service startup time